Sources/Apodini/Configurations/EnvironmentObject.swift

```diff
@@ -0,0 +1,27 @@
+/// A `Configuration` which helps to store and retrieve objects using `@Environment` and the correspond key path.
+///
+/// A warning will be displayed if this action will overwrite a stored property.
+public struct EnvironmentObject<Key: KeyChain, Value>: Configuration {
+    private let value: Value
+    private let keyPath: KeyPath<Key, Value>
+    
+    /// initializer of `EnvironmentObject`.
+    ///
+    /// - Parameters:
+    ///     - value: Object which is stored.
+    ///     - keyPath: Associates a key path conforming to `KeyChain` with the `value`.
+    public init(_ value: Value, _ keyPath: KeyPath<Key, Value>) {
+        self.value = value
+        self.keyPath = keyPath
+    }
+    
+    public func configure(_ app: Application) {
+        if let previousValue = EnvironmentValues.shared.values[ObjectIdentifier(keyPath)] {
+            print("""
+                Warning: A value associated with the key path \(type(of: keyPath)) is already stored.
+                The previous value \(String(describing: previousValue)) will be overwritten with \(String(describing: value)).
+                """)
```

Comment:
I would prefer to have a real logger warning here and not just a print statement. But this is something which is not consistent in Apodini anyway

Reply:
I agree this better than using `print`